assets/src/dashboard/animations/animator/output.js

```diff
@@ -19,21 +19,45 @@
  */
 import PropTypes from 'prop-types';
 
-function WithAnimation({ id, style, children }) {
-  return (
-    <div id={id} style={style}>
+/**
+ * Internal dependencies
+ */
+import { ANIMATION_TYPE } from '../constants';
+
+function WithAnimation({
+  id,
+  animationType,
+  animationStyle,
+  containerStyle,
+  children,
+}) {
+  return animationType === ANIMATION_TYPE.REVEAL ? (
+    <div style={{ overflow: 'hidden', ...containerStyle }}>
+      <div id={id} style={{ width: '100%', height: '100%', ...animationStyle }}>
+        {children}
+      </div>
+    </div>
+  ) : (
+    <div id={id} style={{ ...containerStyle, ...animationStyle }}>
       {children}
     </div>
   );
 }
 
 WithAnimation.propTypes = {
   id: PropTypes.string,
-  style: PropTypes.object,
+  animationType: PropTypes.string.isRequired,
+  animationStyle: PropTypes.object,
+  containerStyle: PropTypes.object,
```

Comment:
I split style into `containerStyle` and `animationStyle` so, if needed, we can apply different styles to the element that's gonna be animated and the container element for the animation.  For most animations (so far), the animation element and container element are the _same_ element.